src/views/tab.rs

```diff
@@ -126,7 +126,9 @@ impl<T> View for Tab<T> {
 
     fn style_pass(&mut self, cx: &mut StyleCx<'_>) {
         for (i, child) in self.id.children().into_iter().enumerate() {
-            cx.style_view(child);
+            if i == self.active {
+                cx.style_view(child);
```

Comment:
I think it's because in `style_view`, there's an early return without calling `self.restore()`.

Reply:
Ahh yes. That was it. Thanks